html_app/scb_ex/addmultipledialog2.soy

```diff
@@ -12,72 +12,62 @@
         Add Samples </h1>
 
     <div class='scb_ex_inner_dialog_body'>
-
-        <table class="scb_s_experiment_setup_table" role='grid' aria-label='Cell Lines'>
-
-
-
-//            {if $assignment.id == 'scb_ex2'}
-
-            <thead class="scb_s_experiment_setup_table_head">
-
-            {foreach $heading in $template.ui.add_multiple_dialog.headings}
-            <td role='columnheader' aria-label='{$heading}' class='scb_s_experiment_setup_table_heading'>
-                {$heading}
-            </td>
-            {/foreach}
+        <table class="scb_s_experiment_setup_table" role='grid' >
```

Comment:
is there a particular reason for the single quotes for `role='grid'` and all the lines after in the HTML tags?